### DISCUSSION TITLE
Collapsable Tab Groups like Chromium.

### DISCUSSION THREAD

**Discussion by unlux** [Floorp-Projects/Floorp#860] [Feature Request]
Having collapsable tab groups is a convenient way to organise our open tabs. Instead of relying on an extension with limited functionality and stability, it would be awesome if we could get this feature natively.

**Comment by BPplays** [comment]
it seems like it's being worked on in upstream Firefox, see [this](https://connect.mozilla.org/t5/ideas/native-tab-grouping-more-customizable-tab-bar/idc-p/63044/highlight/true#M36317)
